### PR TITLE
Feat/space shooter game

### PR DESCRIPTION
**Pull request by shan-226**
This is the basic space shooter game with scoring. Click on the flag icon on the home page's app bar to play the game.

### CODE REVIEW

File: lib/model/space_shooter_game_M/space_shooter_game_model.dart [hunk 1]
```diff
@@ -0,0 +1,75 @@
+// import 'package:flame/components.dart';
```
Comment (MonikaTheiss) [inline lib/model/space_shooter_game_M/space_shooter_game_model.dart:1]: Just a question... why is this all commented out?

**Review by MonikaTheiss — APPROVED** [review]
Looks good , thank you!
Maybe in the future we could look into making the code a bit more modular , but it looks good now.

**Review by pieryvdl — APPROVED** [review]
Looks good. Thank you Shannon! Cool game!

File: lib/model/space_shooter_game_M/space_shooter_game_model.dart (hunk 1 above)
Comment (shan-226) [reply lib/model/space_shooter_game_M/space_shooter_game_model.dart:1]: I changed the structure of my code and didn't end up using that file/code but will clean everything up when the game is done.

**Pull request merged**